dockerfile/ubuntu-20.04-x86.Dockerfile

```diff
@@ -0,0 +1,40 @@
+# Second stage: the actual image
+FROM ubuntu:20.04
+
+ARG DEBIAN_FRONTEND=noninteractive
+ENV GTEST_VERSION=1.13.0
+ENV DOXYGEN_VERSION=1.9.6
+
+# Install build and runtime deps
+COPY /scripts/docker/requirements.txt /opt/tt_metal_infra/scripts/docker/requirements.txt
+RUN apt-get -y update \
+    && xargs -a /opt/tt_metal_infra/scripts/docker/requirements.txt apt-get install -y --no-install-recommends \
+    && rm -rf /var/lib/apt/lists/*
+
+# Install dev deps
+COPY /scripts/docker/requirements_dev.txt /opt/tt_metal_infra/scripts/docker/requirements_dev.txt
+RUN apt-get -y update \
+    && xargs -a /opt/tt_metal_infra/scripts/docker/requirements_dev.txt apt-get install -y --no-install-recommends \
+    && rm -rf /var/lib/apt/lists/*
```

Comment:
Why delete this cache?  IS it for something later?

Reply:
keeps the docker layer smaller.
Realistically this saves us about 50mb